acceptance/acceptance_test.go

```diff
@@ -834,30 +833,55 @@ func getNodeTypeID(cloudEnv string) string {
 
 // buildDatabricksBundlesWheel builds the databricks-bundles wheel and returns the path to the wheel.
 // It's used to cache the wheel build between acceptance tests, because one build takes ~10 seconds.
-func buildDatabricksBundlesWheel(t *testing.T, buildDir string) (string, error) {
+func buildDatabricksBundlesWheel(t *testing.T, buildDir string) string {
+	// Clean up directory, remove all but the latest wheel
+	_ = findWheel(t, buildDir)
+
 	RunCommand(t, []string{"uv", "build", "--no-cache", "-q", "--wheel", "--out-dir", buildDir}, "../experimental/python")
 
-	files, err := os.ReadDir(buildDir)
-	if err != nil {
-		return "", fmt.Errorf("failed to read directory %s: %s", buildDir, err)
+	latestWheel := findWheel(t, buildDir)
+	if latestWheel == "" {
+		t.Errorf("databricks-bundles wheel not found in %s", buildDir)
```

Comment:
This does not call `t.FailNow()`, so it continues execution. Is this intentional?

Reply:
Do we need such fallback? It will be harder to debug if anything goes wrong. CLI commands will also fail in a very weird way

Reply:
https://github.com/databricks/cli/pull/2759/commits/907c258a512013feb579c423a458844c2b83ab34

This will make `DATABRICKS_BUNDLES_WHEEL` undefined and affected bash scripts will fail with clear message because we run them with `-e`.